lib/Word_Lib/Bit_Shifts_Infix_Syntax.thy

```diff
@@ -0,0 +1,209 @@
+(*
+ * Copyright Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ *)
+
+(* Author: Jeremy Dawson, NICTA *)
```

Comment:
github thinks this file is new; sanity check of what happened?

Reply:
It is new, Florian refactored a whole bunch of these. (This is re-import from AFP)